service/matching/handler/engine_integration_test.go

```diff
@@ -1303,58 +1303,6 @@ func (s *matchingEngineSuite) TestConfigDefaultHostName() {
 	s.EqualValues(configEmpty.HostName, "")
 }
 
-func (s *matchingEngineSuite) TestGetTaskListManager_OwnerShip() {
```

Comment:
The unit test itself was failing the go race detector and and was pretty much a pain to work with due to the floating goroutine, so I just refactored it into a non-suite test in membership.go